src/test/java/io/deepstream/ErrorCallback.java

```diff
@@ -0,0 +1,8 @@
+package io.deepstream;
+
+import io.deepstream.constants.Event;
+import io.deepstream.constants.Topic;
+
+public interface ErrorCallback {
```

Comment:
This is great. Much better than testing onTimeout is called. I couldn't think how to do it.